cluster/juju/layers/kubernetes-worker/reactive/kubernetes_worker.py

```diff
@@ -311,9 +311,15 @@ def launch_default_ingress_controller():
     # Render the ingress replication controller manifest
     manifest = addon_path.format('ingress-replication-controller.yaml')
     render('ingress-replication-controller.yaml', manifest, context)
-    kubectl_manifest('create', manifest)
-    hookenv.log('Creating the ingress replication controller.')
-    set_state('kubernetes-worker.ingress.available')
+    if kubectl_manifest('create', manifest):
+        hookenv.log('Creating the ingress replication controller.')
+        set_state('kubernetes-worker.ingress.available')
+        hookenv.open_port(80)
```

Comment:
ok, it makes more sense to have that here when we're taking an action on the ingress LB. Be explicit about opening those ports. +1 THIS LGTM